test/e2e/search_test.go

```diff
@@ -278,7 +281,6 @@ registries = ['{{.Host}}:{{.Port}}']`
 	})
 
 	It("podman search doesn't attempt HTTP if force secure is true", func() {
-		SkipIfRemote("--tls-verify is not supported on podman-remote search")
```

Comment:
@edsantiago I just want to restate what a brilliant change adding this kind of text to the SkipIfRemote function is proving to be.  TYVM once again.